server/test/fixtures/network/nockLicenseVerificationWebsite.post.invalid.html

```diff
@@ -0,0 +1,105 @@
+
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" >
+<HTML>
+	<HEAD>
+		<title>SearchResults</title>
+		<link rel="stylesheet" href="stylesheets/elicense2000.css">
+	</HEAD>
+	<script language="javascript">
+function SetValue(agencyID,licenseID)
+{
+	var href = window.opener.location.href;
+	var form = window.opener.document.forms[0];
+	form.elements["license_id"].value = licenseID;
+	form.elements["tempAgencyID"].value = agencyID;
+//	var action = href.substring(0,href.indexOf("?"));
+//	form.action = action + '?action=lu&agency_id=' + agencyID + '&license_id=' + licenseID;
```

Comment:
Is it safe to remove these commented-out lines?

Reply:
Probably, but these html files represent the data coming back from the CA EMS website. I'd rather leave it in there so that the mock data is an accurate representation of the actual website response.